bblfsh/pyuast.cc

```diff
@@ -954,17 +954,17 @@ static PyObject *PyUastIter_new(PyObject *self, PyObject *args) {
     return nullptr;
 
   // the node can either be external or any other Python object
-  if (PyObject_TypeCheck(obj, &NodeExtType)) {
+  if (PyObject_TypeCheck(obj, &PyNodeExtType)) {
     // external node -> external iterator
-    auto node = (NodeExt*)obj;
+    auto node = (PyNodeExt*)obj;
     return node->ctx->Iterate(obj, (TreeOrder)order);
   }
   // Python object -> create a new context and attach it to an iterator
   Context* ctx = new Context();
   return ctx->Iterate(obj, (TreeOrder)order, true);
 }
 
-static PyObject *PyUastDecode(PyObject *self, PyObject *args) {
+static PyObject *PyContextDecode(PyObject *self, PyObject *args) {
```

Comment:
`Py_Decode`, since it's a global method, or `PyContextExt_decode` since it's a constructor specifically for `PyContextExt`

Reply:
👍 perfect example how this could benefit from new github feature :) https://blog.github.com/category/all/#suggested-changes-public-beta

Reply:
addressed in 2785a60